Content.Shared/Physics/CollisionGroup.cs

```diff
@@ -24,6 +24,7 @@ public enum CollisionGroup
         Underplating    = 1 <<  7, // 128 Things that are under plating
         Passable        = 1 <<  8, // 256 Things that are passable
         ExplosivePassable = 1 << 9, // 512 Things that let the pressure of a explosion through
+        Walls = 1 << 10, // 1024 Special group to "tag" solid walls. Used to check if player can build wallmount on it.
```

Comment:
I'm always suss on adding another layer to the collision group especially for 1 thing (like Passable and ExplosivePassable are). Maybe just do the raycast and pass in a predicate for the wall light tag? @RemieRichards 2nd opinion plz.

Reply:
So, what should I do? Return back to SharedWallComponent?

Reply:
Tags ended up getting merged so go with a tag. Probably raycast and use a predicate on the tag.

The only time collision layers should be added is if they're actually used for collision and not just raycasts.

Reply:
Sadly, tags are server-side right now, so I can't use them in shared namespace for build conditional.
Though after a short discussion tag component will be moved to shared namespace soon.

Reply:
Should be good now as it was merged.